apps/web/src/constants/paths.ts

```diff
@@ -11,4 +11,6 @@ export enum Path {
   Intern = '/admin/intern/:internId',
 
   CatchAll = '/:path*',
+
+  AdminInterviewPicker = '/interview-picker',
```

Comment:
samo molim te promijeni da ima u ruti /admin, i možda /admin/interivew-scheduler

Reply:
i to dodaj gore među admin putanje